render-ws-java-client/src/main/java/org/janelia/render/client/parameter/StackIdWithZParameters.java

```diff
@@ -136,7 +136,7 @@ public List<StackWithZValues> buildListOfStackWithBatchedZ(final RenderDataClien
         return batchedList;
     }
 
-    private List<StackId> getEligibleStackIds(final RenderDataClient renderDataClient)
+    public List<StackId> getEligibleStackIds(final RenderDataClient renderDataClient)
```

Comment:
I know I've been a lot more neglectful than you with commenting in the past, but I suggest we add a doc string if some method's visibility is set to public.

Reply:
agreed - I think it is good practice to javadoc anything public